openedx/core/djangoapps/cors_csrf/models.py

```diff
@@ -17,3 +17,6 @@ class XDomainProxyConfiguration(ConfigurationModel):
             u"requests to this site.  Please list each domain on its own line."
         )
     )
+
+    def __unicode__(self):
```

Comment:
@andy-armstrong: Out of curiosity, I'm wondering what drove this particular override?